### PR TITLE
Add custom config support

### PR DESCRIPTION
**Pull request by cristiandean**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,9 @@ module github.com/globocom/echo-prometheus
 go 1.12
 
 require (
+	github.com/labstack/echo v3.3.10+incompatible
```
Comment (wpjunior) [inline go.mod]: Unused import

File: go.sum [hunk 2]
```diff
@@ -23,10 +24,14 @@ github.com/json-iterator/go v1.1.7/go.mod h1:KdQUCv79m/52Kvf8AW2vK1V8akMuk1QjK/u
 github.com/julienschmidt/httprouter v1.2.0/go.mod h1:SYymIcj16QtmaHHD7aYtjjsJG7VTCxuUUipMqKk8s4w=
 github.com/konsorten/go-windows-terminal-sequences v1.0.1/go.mod h1:T0+1ngSBFLxvqU3pZ+m/2kptfBszLMUkC4ZK/EgS/cQ=
 github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515/go.mod h1:+0opPa2QZZtGFBFZlji/RkVcI2GknAs/DXo4wKdlNEc=
+github.com/labstack/echo v3.3.10+incompatible h1:pGRcYk231ExFAyoAjAfD85kQzRJCRI8bbnE7CX5OEgg=
```
Comment (wpjunior) [inline go.sum]: Unused import, please run 'go mod tidy'

File: middleware.go [hunk 3]
```diff
@@ -1,45 +1,46 @@
...
+		0.005,
+		0.01, // 10ms
+		0.02,
+		0.05,
+		0.1, // 100 ms
+		0.2,
+		0.5,
+		1.0, // 1s
+		2.0,
+		5.0,
```
Comment (wpjunior) [inline middleware.go]: Can we add 10.0 15.0, 20.0 and 30.0 buckets?

**Comment by cristiandean** [comment]
Done @wpjunior

**Comment by wpjunior** [comment]
Thanks @cristiandean =D

**Pull request merged**